.claude/skills/test-writer/references/fixture-guide.md

```
@@ -0,0 +1,53 @@
+# Fixture & Fake 구현체 가이드
+
+## Fixture 사용 원칙
+
+### 기존 Fixture 활용
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)